src-ui/res/rules.ja.yaml

```diff
@@ -262,3 +262,4 @@ bosnianroad: "1. いくつかのマスを黒く塗り、黒マスでひとつの
 subomino: "1. 点線上に線を引き、盤面の白マスをブロックに分割する。\n2. 数字は、そのマスを含むブロックのマス数を表す。ブロックは数字をいくつ含んでも良い。\n3. 辺で接するブロック同士は、一方が他方を包含してはいけない。ただし包含とは、片方のブロックを回転せずに平行移動させたとき、もう片方のブロックの上にマスがはみ出ず重ねられることである。"
 trainstations: "1. 盤面の全てのマスにタテヨコに線を引き、全体で１つの輪っかを作ります。\n2. 輪っかを作る線はマスの中央を通り、盤面の外に出たり、枝分かれしたり、行き止まりがあったりしてはいけません。\n3. ┼のマスでは、線は必ず立体交差します。┼以外のマスで線が交差してはいけません。\n4. 数字は駅を表します。駅は、数字の順番通りに通る必要があります。\n5. 線は駅のマスの上で直進します。"
 lineofsight: "1. パズル面のいくつかの頂点を線でつないで枝分かれのない一つのループを作る。\n2. 数字と矢印は、その方向を見たときに見える直線の長さを表す。"
+turaround: "1. 盤面にタテヨコに線を引き、全体で１つの輪っかを作ります\n2. 輪っかを作る線はマスの中央を通り、盤面の外に出たり、交差したり、枝分かれしたり、行き止まりがあったりしてはいけません\n 3. 線はすべての数字の上を通ります\n4. 数字はその数字マスと2つの隣のマス線が曲がる回数を表します。"
```

Comment:
Was this written using Google Translate or another machine translation? It's better to remove it than to add Japanese text that we can't probably correct.